app/views/recipes/public_recipe.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="body-max-width mx-auto mt-5 ">
+  <div class="container">
+    <div class="card flat-ui-card shadow-lg">
+      <div class="card-header d-flex justify-content-between align-items-center flat-ui-header">
+        <div class="d-flex align-items-center">
+          <i class="bi bi-book me-2 fs-4"></i>
+          <h4 class="m-0">Public Recipes</h4>
+        </div>
+        <%= link_to '+ Add New Recipe', new_recipe_path, class: 'btn btn-primary' %>
+      </div>
+      <div class="card-body">
+        <% if @recipes.any? %>
+          <div class="row g-4">
+            <% @recipes.each do |recipe| %>
+              <div class="col-12 col-md-6">
+                <div class="card recipe-card flat-ui-recipe-card">
+                  <div class="card-body">
+                    <h5 class="card-title"><%= link_to recipe.name, recipe, class: 'text-decoration-none' %></h5>
+                    <p class="card-text"><%= recipe.description %></p>
+                  </div>
+                  <% if current_user == recipe.user %>
+                  <div class="card-footer">
+                    <button type="button" class="btn btn-danger btn-sm" data-bs-toggle="modal" data-bs-target="#deleteRecipeModal" data-recipe-id="<%= recipe.id %>">
+                      Delete
+                    </button>
+                  </div>
+                
+                  <div class="modal fade" id="deleteRecipeModal" tabindex="-1" aria-labelledby="deleteRecipeModalLabel" aria-hidden="true">
+                    <div class="modal-dialog modal-dialog-centered">
+                      <div class="modal-content">
+                        <div class="modal-header">
+                          <h5 class="modal-title" id="deleteRecipeModalLabel">Confirm Deletion</h5>
+                          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                        </div>
+                        <div class="modal-body">
+                          Are you sure you want to delete this recipe?
+                        </div>
+                        <div class="modal-footer">
+                        <%= button_to 'Delete', recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger' %>
+                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
+                      </div>                      
+                      </div>
+                    </div>
+                  </div>
+                <% end %>
+                
+                </div>
+              </div>
+            <% end %>
+          </div>
+        <% else %>
+          <p class="text-center">You haven't created any recipes yet.</p>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Based on the wireframe and project requirements, the **_Public Recipe page_** should contain the owner of the recipe, the total food items needed to prepare the recipe and the total cost of the recipe. Kindly include this on your page. 👍 😎 

### Wireframe

![image](https://github.com/Diegogagan2587/Recipe-app-bests-dishes/assets/77106962/41e10bea-f2a9-44d0-aff9-06fe1978054e)

### Your Design

![image](https://github.com/Diegogagan2587/Recipe-app-bests-dishes/assets/77106962/d81b4c04-417d-4527-8c06-52160a41a377)

#### N.B: On the **_Public recipe page_**, there shouldn't be any button to add ingredient. Kindly implement this. 👍 😎